fem/integ/bilininteg_mass_kernels.hpp

```diff
@@ -31,6 +31,31 @@ void PAMassAssembleDiagonal(const int dim, const int D1D,
                             const Vector &D,
                             Vector &Y);
 
+inline void PAMassAssembleDiagonal1D(const int NE,
```

Comment:
Just because it isn't templated and can be, should this definition go in `bilininteg_mass_kernels.cpp`? No real preference here but just wanted to call it out.

Also would be nice to add a short comment for consistency: `// PA Mass Diagonal 1D kernel`.

Reply:
Sure, why not. Done in 4438077f6